src/RunState.h

```diff
@@ -70,6 +70,7 @@ extern void suspend_processing();
 extern void continue_processing();
 bool is_processing_suspended();
 
+[[deprecated("Remove with v8.1. Use run_state::current_pseudo directly, but you probably should not")]]
 extern double current_packet_timestamp();
```

Comment:
Let me just double check: Iirc, it's called `current_packet_timestamp()` because it used to give you whatever semantically was the current timestamp, either the actual time of a live packet or the pseudo time in pseudo-realtime-mode. In other words, the caller didn't need to worry about which mode we're in. 

That is no longer the case, correct? I see that it now always returns `current_pseudo`  (as you say) and I assume that's not set to anything reasonable when not in PRM?

Reply:
> That is no longer the case, correct?

Correct and I actually don't see when it ever was - 69da2d7b1d215832f1d948cefb4d01e7ea698a03.

And yes, current_pseudo is only set with pseudo realtime. So I figured I'd just remove the generic accessor. `current_packet_wallclock()` is similarly, but it does more that I din't dare to touch for now.